pkg/ccl/crosscluster/replicationutils/utils.go

```diff
@@ -323,3 +323,31 @@ func TestingGetPTSFromReplicationJob(
 
 	return ptsRecord.Timestamp
 }
+
+func TestingGetRetainedTimeFromReplicationJob(
+	t *testing.T,
+	ctx context.Context,
+	sqlRunner *sqlutils.SQLRunner,
+	srv serverutils.ApplicationLayerInterface,
+	producerJobID int,
```

Comment:
you're trying to grab the pts from the consumer job, right? TestingGetPTSFromReplicationJob grabs the pts on the producer job already.